src/esp/gfx/PTexMeshShader.h

```diff
@@ -23,19 +23,51 @@ namespace gfx {
 
 class PTexMeshShader : public Magnum::GL::AbstractShaderProgram {
  public:
+  //! @brief vertex positions
   typedef Magnum::GL::Attribute<0, Magnum::Vector4> Position;
 
+  /**
+   * @brief Constructor
+   */
   explicit PTexMeshShader();
 
   // ======== texture binding ========
+  /**
+   *! @brief Bind the atlas texture
+   *! @return Reference to self (for method chaining)
+   */
   PTexMeshShader& bindAtlasTexture(Magnum::GL::Texture2D& texture);
+  /**
+   *! @brief Bind the buffer texture containing the adjacent faces
+   *! @return Reference to self (for method chaining)
+   */
   PTexMeshShader& bindAdjFacesBufferTexture(Magnum::GL::BufferTexture& texture);
 
   // ======== set uniforms ===========
+  /**
+   *! @brief Set modelview and projection matrix to the uniform on GPU
+   *! @return Reference to self (for method chaining)
+   */
```

Comment:
Can we maybe decide on a consistent docstring style? :) It's `//!` on some places, #195 uses `/**` and here it's `*!`. (Actually, does Doxygen even understand these without putting exclamation marks to the output? I know only of `/*!`, but not `*!`).

Cc: @aclegg3 @msavva @erikwijmans

Reply:
@mosra :
Are you commenting on the latest version? I did not see exclamation marks on my side (no new version was pushed after your comments).
The docstrings in this diff was following the format in the PrimitiveIDShader.h

Reply:
![image](https://user-images.githubusercontent.com/931093/64877982-59219900-d607-11e9-85c5-bc14cec97bdb.png)

Reply:
Uh oh, sorry, GitHub is acting up and showing it like this in the conversation here

![image](https://user-images.githubusercontent.com/344828/64879891-42a32e00-d657-11e9-9ddd-bd8689e6686b.png)

but when I open the file diff, those are not there anymore. Not sure what's up. Apologies for the noise.

Reply:
@mosra : NO worries! Sometimes GitHub behaves a bit abnormal. I totally understand since I vaguely remembered I had similar experience before.